v2.0/restore.md

```diff
@@ -109,8 +109,13 @@ The URL for your backup's locations must use the following format:
 | Google Cloud Storage | `gs` | Bucket name | None––currently only supports instance auth, but we can build non-instance auth at a customer's request |
 | HTTP | `http` | Remote host | N/A |
 | NFS | `nodelocal` | File system location | N/A |
+| S3-compatible services | `s3` | Bucket name | `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION`, `AWS_ENDPOINT` |
 
-{{site.data.alerts.callout_info}}Backups stored on NFSes work only if each node in the cluster accesses the drive in the same way.{{site.data.alerts.end}}
+#### Considerations
+
+- The backup location parameters often contain special characters that need to be URI-encoded. Use Javascript's [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) function or Go language's [url.QueryEscape](https://golang.org/pkg/net/url/#QueryEscape) function to URI-encode the parameters. Other languages provide similar functions to URI-encode special characters.
+- Backups stored on NFSes work only if each node in the cluster accesses the drive in the same way.
```

Comment:
@dt Please review this section

Reply:
The `--external-io-dir` mitigates this limitation to some extent -- previously, since the path was relative to `/`, every node had to mount the NFS share in the same place, like `/mnt/nfs/backups` or whatever. Now that the backup path is relative to the node's configured external-io-dir, nodes can have the NFS mount at whatever location they want as long as they all point to the same share using their `external-io-dir` flag.. e.g. one node might be started with `--external-io-dir=/mnt/nfs/backups` while another might use `--external-io-dir=/Volumes/backups` or whatever.

The important thing is that all nodes use `--external-io-dir` to point to the same share.

Reply:
@dt So I can delete the second bullet (Backups stored on NFSes...), right?

Reply:
@Amruta-Ranade, @dt, has this conversation been resolved?